src/Compilers/CSharp/Test/Semantic/Semantics/SpanStackSafetyTests.cs

```diff
@@ -1131,7 +1132,7 @@ public class Program
 {
     static void Main()
     {
-        // we allow this. Is that because it would be a breaking change?
+        // we no longer allow this.
```

Comment:
Please update the breaking changes doc as part of the PR.

Reply:
The breaking change doc has been updated.

---
In reply to: [149494899](https://github.com/dotnet/roslyn/pull/23055#discussion_r149494899) [](ancestors = 149494899)